Wikibase.php

```diff
@@ -108,6 +108,20 @@
 		// hardcoded internal handling
 		'html',
 	];
+	$wgWBRepoSettings['entityDataCachePaths'] = function() {
+		return [
+			// JSON from entity page JS, compare wikibase.entityPage.entityLoaded.js
+			wfAppendQuery(
+				str_replace( '$1', 'Special:EntityData/{entity_id}.json', $GLOBALS['wgArticlePath'] ),
+				'revision={revision_id}'
+			),
+			// Turtle from Query Service updater
+			wfAppendQuery(
+				str_replace( '$1', 'Special:EntityData/{entity_id}.ttl', $GLOBALS['wgArticlePath'] ),
+				'flavor=dump&revision={revision_id}'
+			),
+		];
+	};
```

Comment:
Not sure the ; is needed

Reply:
okay

Reply:
welp, as it stands, I think it is needed